### PR TITLE
MANTA-5272 minnow agent build fails for mantav1 mako due to restdown py3

### PR DESCRIPTION
**Pull request by timfoster**

### CODE REVIEW

**Comment by timfoster** [comment]
The build is failing at the moment because we're trying to build the current mantav1 branch of manta-minnow on a python3-equipped build machine. Once the minnow PR lands, this build should be clean.

File: Makefile [hunk 1]
```diff
@@ -167,3 +160,9 @@ ifeq ($(shell uname -s),SunOS)
 endif
 include ./tools/mk/Makefile.nginx.targ
 include ./deps/eng/tools/mk/Makefile.targ
+
```
Comment (trentm) [inline Makefile:163]: Was there a specific need to move this block lower in the Makefile?

Comment (timfoster) [reply Makefile:163]: Yes, that was intentional. We need to have this after the include of Makefile.agent_prebuilt.defs because otherwise `$(MINNOW_PREBUILT_TARGETS)` isn't defined, so we end up setting a variable for a null target.

In action, that old behaviour resulted in this - note the incorrect `-b` option, despite the fact we did have that `$(MINNOW_PREBUILT_TARGETS)...` line at 67:

```
timf@jenkins-agent-x86_64-19.4.0 (prr-MANTA-5272) make -f Makefile.old MINNOW-prebuilt-extract
mkdir -p /tmp/buildimage-mako-prr-MANTA-5272-20200603T135010Z-g1ef00e1; \
cd /tmp/buildimage-mako-prr-MANTA-5272-20200603T135010Z-g1ef00e1; \
/mnt/tank/projects/manta-mako.git/deps/eng/tools/agent-prebuilt.sh \
        -b 'prr-MANTA-5272' \
        -B '' \
        -c '/var/tmp/agent-cache.timf' \
        -d '' \
        -p 'minnow-pkg-.*.tar.gz' \
        -r 'manta-minnow' \
        -U '' extract
Extracting agent /var/tmp/agent-cache.timf/manta-minnow/.//minnow-pkg-master-20200422T134809Z-g68eb411.tar.gz to /tmp/buildimage-mako-prr-MANTA-5272-20200603T135010Z-g1ef00e1
timf@jenkins-agent-x86_64-19.4.0 (prr-MANTA-5272)
```

rather than what we want, moving the declaration down a bit:

```
timf@jenkins-agent-x86_64-19.4.0 (prr-MANTA-5272) make MINNOW-prebuilt-extract
mkdir -p /tmp/buildimage-mako-prr-MANTA-5272-20200603T135002Z-g1ef00e1; \
cd /tmp/buildimage-mako-prr-MANTA-5272-20200603T135002Z-g1ef00e1; \
/mnt/tank/projects/manta-mako.git/deps/eng/tools/agent-prebuilt.sh \
        -b 'mantav1' \
        -B '' \
        -c '/var/tmp/agent-cache.timf' \
        -d '' \
        -p 'minnow-pkg-.*.tar.gz' \
        -r 'manta-minnow' \
        -U '' extract
Extracting agent /var/tmp/agent-cache.timf/manta-minnow/.//minnow-pkg-master-20200422T134809Z-g68eb411.tar.gz to /tmp/buildimage-mako-prr-MANTA-5272-20200603T135002Z-g1ef00e1
```

basically, breaking development builds where we would try to find the `prr-MANTA-5272` branch of the agent (matching the branch name of the mako bits) before falling back to `master`

What we want for mantav1 builds, is to always use the `mantav1` branch of the agent.

Comment (timfoster) [reply Makefile:163]: I should say, "production" builds were unaffected by this bug. If we built manta-mako.git on the `mantav1` branch, that branch value would be passed to the `-b` option of agent-prebuilt.sh, and everything was fine because we'd then build a mantav1 branch of the agent, which was the desired outcome.
